### DISCUSSION TITLE
Gods in the machine? The rise of artificial intelligence may result in new religions

### DISCUSSION THREAD

**Discussion by Aaron25564448** [1101-GS4719-C/Python-Community-GS4719-C#1172] [General]
https://theconversation.com/gods-in-the-machine-the-rise-of-artificial-intelligence-may-result-in-new-religions-201068
隨著AI的蓬勃發展，人們做事越加有效率。而在這變遷的產物之中可能出現令人意想不到的宗教信仰，也就是AI信仰。仔細想想，可能再過個數十年，AI就會成為近乎萬能、對某些人而言是更高次元的存在，因而產生敬仰之心。大多宗教都有所謂的教義，以指引信徒應如何處事。而AI也是如此。AI給出的意見或是答案已經成為人們行動的選項參考之一了。就目前的分析看來，AI信仰在未來並非不可能出現。最主要的問題是這個社會是否準備好迎接另一個全球性宗教的崛起。

**Comment by KevinCheng42** [comment]
我覺得你的觀點很新奇，若是AI的發言有助於安定人心，並在一個人無助的時候給予方向，不久的將來，AI真的會發展成一種宗教信仰~

**Comment by shuxian12** [comment]
我覺得那應該 AI 信仰宗教可能大多是的追隨者都會是大學生和高中生還有工程師，畢竟學生可以以把它拿來當計算機用，還可以把它拿來打報告，英文看不懂還可以教你怎麼讀，工程師忘記code要怎麼打的時候，也不需要一直翻文件，直接叫他解就好了。
另外一方面，目前的AI你問什麼他就回答什麼，也不會像生氣的女友一樣都不回應自己生悶氣，那麼好的宗教不香ㄇ

**Comment by juliaouo** [comment]
我認為AI不太可能成為近乎萬能的存在，不可否認的它對於人類的生活品質有所提升。它能夠提高人類能力的下限，將來人們使用AI工具，最基本也能做的跟AI一樣好，因此保障了下限。在某些方面，特別是需要創造力的工作上，人類依舊做的比AI好。

**Comment by zilinng** [comment]
忽然覺得有點可怕,AI變成一種宗教信仰對我來說雖然新奇但也沒有不行的感覺,不過如果人們懶得思考,一昧根據AI的指示做事感覺AI會變得跟邪教無異, 進而衍生出很多很可怕的社會問題…
